test/e2e/storage/testsuites/snapshottable.go

```diff
@@ -269,9 +269,17 @@ func (s *snapshottableTestSuite) DefineTests(driver storageframework.TestDriver,
 				framework.ExpectNoError(err)
 
 				ginkgo.By("should delete the VolumeSnapshotContent according to its deletion policy")
-				err = storageutils.DeleteAndWaitSnapshot(dc, vs.GetNamespace(), vs.GetName(), framework.Poll, f.Timeouts.SnapshotDelete)
+
```

Comment:
Is DeleteAndWaitSnapshot used in any other tests? Will those other tests have the same issue?

Reply:
It's being used in the csi_mock_volume test and It seems the mock driver is okay with the ordering.